src/types/ProposalFieldValue.ts

```diff
@@ -38,6 +41,14 @@ export const proposalFieldValueSchema: JSONSchemaType<ProposalFieldValue> = {
       required: [],
       instanceof: 'Date',
     },
+    applicationFormField: {
+      type: 'object',
+      /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+      properties: applicationFormFieldSchema.properties,
+      /* eslint-enable @typescript-eslint/no-unsafe-assignment */
```

Comment:
I would really like to figure out a way to include the application form field schema without having to disable an `eslint` rule. This seemed to me the least worst option that I could come up with. I am eager to hear about better ways to achieve the goal, though.

Reply:
You can actually just say:

```
applicationFormField: applicationFormFieldSchema
```

Which should solve your problem AND avoid some repeated code!

Reply:
That is much more elegant! Yes!

Unfortunately it doesn't "just work" without some more finagling. I'll figure out what else needs to be updated to get it to work.